bundle/edu.gemini.ags/src/main/scala/edu/gemini/ags/gems/GemsResultsAnalyzer.scala

```diff
@@ -130,10 +130,10 @@ object GemsResultsAnalyzer {
   }
 
   private def printResults(result: List[GemsGuideStars]) {
-    Log.info("Results:")
-    result.zipWithIndex.foreach { case (s, i) =>
-      Log.info(s"result #$i : $s")
-    }
+    // Log.info("Results:")
+    // result.zipWithIndex.foreach { case (s, i) =>
+    //   Log.info(s"result #$i : $s")
+    // }
```

Comment:
One idea for this would be to make them `Log.fine("...")` instead of commenting them out.  Then @andrewwstephens could lower the log level in his OT and see all the minute details.